modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/DetectorMappingController.java

```diff
@@ -41,7 +41,7 @@
     private DetectorMappingService detectorMappingService;
 
     @RequestMapping(produces = "application/json", method = RequestMethod.GET)
-    public DetectorMapping getDetectorMapping(@RequestParam String id) {
+    public ElasticsearchDetectorMapping getDetectorMapping(@RequestParam String id) {
```

Comment:
Why does the controller know about the backend data store? Shouldn't Elasticsearch be invisible to the controller?

Reply:
We still have detector mapping entity for mysql and here I have just renamed detector mapping entity to ElasticsearchDetectorMapping and put it inside entity package. Once we get rid of mysql, we can fix it.